### PR TITLE
Add Elixir 1.18 to allowed Elixir versions

### LINKED ISSUE #846: Elixir 1.18 support
Lexical should support 1.18.

### PR DESCRIPTION
**Pull request by kevinschweikert**
Closes #846

### CODE REVIEW

File: apps/server/lib/lexical/server/boot.ex [hunk 1]
```diff
@@ -88,7 +88,8 @@ defmodule Lexical.Server.Boot do
     "1.15.0" => ">= 1.15.3",
     "1.16.0" => ">= 1.16.0",
     "1.17.0-rc" => ">= 1.17.0-rc",
-    "1.17.0" => ">= 1.17.0"
+    "1.17.0" => ">= 1.17.0",
+    "1.80.0" => ">=1.18.0"
```
Comment (sorentwo) [inline apps/server/lib/lexical/server/boot.ex:92]: Typo here. We have a long while until v1.80!

```suggestion
    "1.18.0" => ">= 1.18.0"
```

**Comment by sorentwo** [comment]
Thanks for the patch. Happy to report that with this patch (and the small correction I commented about), Lexical works normally with Elixir v1.18.

File: apps/server/lib/lexical/server/boot.ex (hunk 1 above)
Comment (kevinschweikert) [reply apps/server/lib/lexical/server/boot.ex]: Thanks @sorentwo for the correction! Must have been wishful thinking 😉

**Comment by scohen** [comment]
There's more that needs to be done to get lexical to properly support 1.18. Closing in favor of #849

**Pull request closed without merging**